lib/forwarder.js

```diff
@@ -12,15 +12,26 @@ import { removeConfig } from '../lib/config.js';
  * @param {Config} config
  */
 export async function forwardToDaemon(resolver, config) {
+  const eslintArgs = process.argv.slice();
```

Comment:
May I ask you to follow the naming convention for variables that is used in the rest of the codebase?

- `camelCase` for functions
- `snake_case` for values

Reply:
Which world would we be leaving in if you couldn't? That would definitely be chaos :-).

Done (and sorry for not paying attention) (and it would be good if there was an eslint rule for that, maybe https://github.com/ptkdev/eslint-plugin-snakecasejs but it seems unmaintained)